### ISSUE TITLE
Right controller movement prioritized over left controller movement

### ISSUE THREAD

**Issue by Armored-Dragon** [overte-org/overte#728]
When playing in game in VR, your player turning is prioritized over your character movement controls or something like that.

Steps to reproduce:
1: Enter the game
2: Move your right thumbstick (Which would turn you) any direction, including up or down (Which won't turn you). 
3: Attempt to move your character in any direction.

You can only move side to side when your right thumbstick is in use.

**Comment by SilverfishVR** [comment]
This could be related to #643 where the teleport action (right stick up) gets activated because it has zero deadzone.

If that is the case you should, in theory, be able turn and move at the same time if you are some kind of ninja that can move the stick perfectly in the X axis only.

**Comment by Armored-Dragon** [comment]
> This could be related to #643 where the teleport action (right stick up) gets activated because it has zero deadzone.
> 
> If that is the case you should, in theory, be able turn and move at the same time if you are some kind of ninja that can move the stick perfectly in the X axis only.

I was experiencing this issue with Teleport mode completely disabled. Unless there is hard-coded functionality that does not properly check to see if teleport mode is enabled before running, it may not be strictly related.

**Comment by SilverfishVR** [comment]
Yes, the controller action for teleport is active even if you have teleport disabled.
